content/help/ops/traffic-management/troubleshooting/index.md

```diff
@@ -6,6 +6,39 @@ weight: 30
 
 This section describes common problems and tools and techniques to address issues related to traffic management.
 
+## Requests are rejected by Envoy
+
+Requests may be rejected for various reasons, to identify the reasons, and you may inspect
```

Comment:
Please update the start of the paragraph:

Requests may be rejected for various reasons. The best way to understand why requests are being rejected is by inspecting Envoy's access logs. By default, access logs...

Reply:
Done